tests/unit/routes/index-test.js

```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:index', 'Unit | Route | index', {
```

Comment:
So most of the routes will be tested in acceptance tests. There are few occasions that you will test it by itself (see docs if you want to know more). Also just delete the useless tests